text/0000-path-clarity.md

```diff
@@ -8,12 +8,12 @@
 
 This RFC seeks to clarify and streamline Rust's story around paths and visibility for modules and crates. When the RFC is *fully* implemented (which will require a [new epoch]), that story will look as follows:
 
-- Absolute paths *always* begin with a crate name, where the keyword `crate` refers to the current crate.
+- Absolute paths begin with a crate name, where the keyword `crate` refers to the current crate (other forms are deprecated)
```

Comment:
Can there be a built in lint for the older forms? Doesn't have to be warn by default, can also be allow by default.